src/wrappers/_MarkdownWrapper.jsx

```diff
@@ -1,65 +1,47 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import TerraMarkdown from 'terra-markdown';
 import classNames from 'classnames/bind';
+
+import CodesplitWrapper from './_CodesplitWrapper';
 import ContentLoaded from './_ContentLoaded';
 import ContentLoading from './_ContentLoading';
-import ErrorPage from './_ErrorPage';
 
-import styles from './Markdown.module.scss';
+import styles from './MarkdownWrapper.module.scss';
 
 const cx = classNames.bind(styles);
-
 const propTypes = {
   /**
    * The content to be placed within the main content area of the container.
    */
-  content: PropTypes.func.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  content: PropTypes.object.isRequired,
+  /**
+   * The props to be applied to the content.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  props: PropTypes.object,
 };
 
-const MarkdownWrapper = ({ content }) => {
-  const [state, setState] = useState({
-    markdown: undefined,
-    isErrored: false,
-  });
-  const { markdown, isErrored } = state;
-
-  // Dynamically import Javascript chunk
-  useEffect(() => {
-    let isActive = true;
-    if (!markdown && !isErrored) {
-      content().then(({ default: markdownFile }) => {
-        if (isActive) {
-          setState({
-            markdown: markdownFile,
-            isErrored: false,
-          });
-        }
-      }).catch(() => {
-        if (isActive) {
-          setState({
-            markdown: undefined,
-            isErrored: true,
-          });
-        }
-      });
-    }
-    return () => { isActive = false; };
-  });
+const defaultProps = {
+  props: undefined,
+};
 
-  if (markdown || isErrored) {
-    return (
+const MarkdownWrapper = (props) => (
+  <CodesplitWrapper
+    {...props}
+    fallback={<ContentLoading />}
+    loadedWrapper={({ children }) => (
       <ContentLoaded>
-        { markdown
-          ? <div className={cx('markdown')}><TerraMarkdown src={markdown} /></div>
-          : <ErrorPage error="The page failed to load. Refresh the page to try again." />}
+        <div className={cx('markdown')}>
+          {children}
+        </div>
       </ContentLoaded>
-    );
-  }
-
-  return <ContentLoading />;
-};
+    )}
+    errorWrapper={ContentLoaded}
```

Comment:
Should this be the ErrorPage?

Reply:
No, this just wraps the error page with the appropriate div's with classnames for testing.